src/lib/components/Breadcrumb.svelte

```diff
@@ -0,0 +1,36 @@
+<script>
+	export let links;
+</script>
+
+<div>
+	<ol>
```

Comment:
It looks like we accidentally moved from 2 space indents to 4 space?

Reply:
I don't think I actively changed any prettier rules, however I did opt in to use SvelteKit's prettier setting so it might have messed something up. I'll investigate